redash/query_runner/__init__.py

```diff
@@ -208,6 +219,22 @@ def _get_tables_stats(self, tables_dict):
                 res = self._run_query_internal("select count(*) as cnt from %s" % t)
                 tables_dict[t]["size"] = res[0]["cnt"]
 
+    @property
+    def supports_auto_limit(self):
+        return True
+
+    def apply_auto_limit(self, query_text, set_auto_limit):
+        if set_auto_limit:
+            from redash.query_runner.databricks import split_sql_statements
+            query_list = split_sql_statements(query_text)
```

Comment:
How about renaming this to `queries`? Otherwise it feels a bit like `user_name_string`.